starknet_py/net/gateway_client.py

```diff
@@ -7,26 +7,28 @@
 from starkware.starknet.services.api.gateway.transaction import AccountTransaction
 
 from starknet_py.net.client import Client
-from starknet_py.net.client_models import (
-    Transaction,
-    SentTransactionResponse,
-    ContractCode,
+
+from starknet_py.net.models.contracts import ContractCode, DeclaredContract, Call
+from starknet_py.net.models.blocks import (
+    GatewayBlock,
+    BlockTransactionTraces,
     BlockStateUpdate,
-    InvokeFunction,
-    StarknetTransaction,
     Hash,
     Tag,
-    DeclaredContract,
+)
+from starknet_py.net.models.transaction import (
+    Transaction,
```

Comment:
Using `Transaction` for both making and receiving transactions will cause problems. That's why it was named `StarknetTransaction` before (though it's not an ideal name).

Also this renaming is kind of breaking?

Reply:
Problem is parts of starknet.py se `Transaction` and parts use `StarknetTransaction` for the same object, so it's unclear to me how can we resolve this.